userland/init/init.c

```diff
@@ -24,6 +24,7 @@ void uptime() {
 }
 
 int main() {
+    
```

Comment:
I'll revert this change as well.

Reply:
Are you sure about that one? Though it'd be nice to have a style guide or contribution guide of sorts.

I find that a few newlines and spaces can really improve the code readability overall.

Reply:
> I find that a few newlines and spaces can really improve the code readability overall.

I kinda agree in general but I don't like newlines below function names.